diff --git a/libpdbg/dtb.c b/libpdbg/dtb.c
index b094e34..efed426 100644
--- a/libpdbg/dtb.c
+++ b/libpdbg/dtb.c
@@ -56,6 +56,7 @@
 #define CHIP_ID_P8P 0xd3
 #define CHIP_ID_P9  0xd1
 #define CHIP_ID_P9P 0xd9
+#define CHIP_ID_P10 0xda
 
 static enum pdbg_backend pdbg_backend = PDBG_DEFAULT_BACKEND;
 static const char *pdbg_backend_option;
@@ -212,6 +213,7 @@ static void bmc_target(struct pdbg_dtb *dtb)
 	switch(chip_id) {
 	case CHIP_ID_P9:
 	case CHIP_ID_P9P:
+	case CHIP_ID_P10:
 		pdbg_log(PDBG_INFO, "Found a POWER9 OpenBMC based system\n");
 		if (!dtb->backend.fdt)
 			dtb->backend.fdt = &_binary_p9_kernel_dtb_o_start;
