commit b56659b5c99a0aa3b8ca0e8758818e88552c0136
Author: Ramesh Iyyar <rameshi1@in.ibm.com>
Date:   Sat Nov 30 01:21:52 2019 -0600

    fapi2: Add missing include file required for memcpy()

diff --git a/ekb/hwpf/fapi2/include/collect_reg_ffdc.H b/ekb/hwpf/fapi2/include/collect_reg_ffdc.H
index beeecfb..a8a2711 100644
--- a/ekb/hwpf/fapi2/include/collect_reg_ffdc.H
+++ b/ekb/hwpf/fapi2/include/collect_reg_ffdc.H
@@ -31,6 +31,7 @@
 #include <plat_trace.H>
 #include <endian.h>
 
+#include <string.h>
 
 namespace fapi2
 {
