diff --git a/Makefile.istep0 b/Makefile.istep0
index e69de29..9b0ea4f 100644
--- a/Makefile.istep0
+++ b/Makefile.istep0
@@ -0,0 +1,5 @@
+# istep0
+include ekb/chips/p10/procedures/hwp/perv/p10_start_cbs.mk
+
+# istep6-21
+include ekb/chips/p10/procedures/hwp/istep/p10_do_fw_hb_istep.mk
diff --git a/Makefile.generate b/Makefile.generate
index c6ef9e8..d4c9f28 100644
--- a/Makefile.generate
+++ b/Makefile.generate
@@ -112,7 +112,7 @@ endef
 
 all: definitions
 
-include Makefile.istep
+include Makefile.istep0
 
 SLIST += $(SRC_EXTRA)
 
