diff --git a/libipl/p10/ipl20.C b/libipl/p10/ipl20.C
index 5d28e78..6465df2 100644
--- a/libipl/p10/ipl20.C
+++ b/libipl/p10/ipl20.C
@@ -18,8 +18,14 @@ static int ipl_host_load_payload(void)
 	return ipl_istep_via_hostboot(20, 1);
 }
 
+static int ipl_host_load_complete(void)
+{
+    return ipl_istep_via_hostboot(20, 2);
+}
+
 static struct ipl_step ipl20[] = {
 	{ IPL_DEF(host_load_payload),   20,  1,  true,  true  },
+	{ IPL_DEF(host_load_complete),   20,  2,  true,  true  },
 	{ NULL, NULL, -1, -1, false, false },
 };
 
diff --git a/libipl/p10/ipl21.C b/libipl/p10/ipl21.C
index 7599561..d3ef3a7 100644
--- a/libipl/p10/ipl21.C
+++ b/libipl/p10/ipl21.C
@@ -15,7 +15,7 @@ static void ipl_pre21(void)
 
 static int ipl_host_runtime_setup(void)
 {
-	return ipl_istep_via_hostboot(21, 1);
+	return ipl_istep_via_hostboot(21, 2);
 }
 
 static int ipl_host_verify_hdat(void)
@@ -29,7 +29,7 @@ static int ipl_host_start_payload(void)
 }
 
 static struct ipl_step ipl21[] = {
-	{ IPL_DEF(host_runtime_setup),        21,  1,  true,  true  },
+	{ IPL_DEF(host_runtime_setup),        21,  2,  true,  true  },
 	{ IPL_DEF(host_verify_hdat),          21,  3,  true,  true  },
 	{ IPL_DEF(host_start_payload),        21,  4,  true,  true  },
 	{ NULL, NULL, -1, -1, false, false },
